example/stochastic_subgrid/stochastic_subgrid_driver.cpp

```diff
@@ -161,8 +161,7 @@ TaskCollection StochasticSubgridDriver::MakeTaskCollection(BlockList_t &blocks,
           tl.AddTask(update, parthenon::cell_centered_bvars::ReceiveBoundBufs<any>, mc1);
       auto set = tl.AddTask(recv, parthenon::cell_centered_bvars::SetBounds<any>, mc1);
       if (pmesh->multilevel) {
-        tl.AddTask(set, parthenon::cell_centered_refinement::RestrictPhysicalBounds,
-                   mc1.get());
+        tl.AddTask(set, parthenon::cell_centered_bvars::RestrictGhostHalos, mc1, false);
```

Comment:
I believe the original naming was misleading---and it was my fault---I discovered the need to do the restriction because of physical boundaries. However, we need to restrict all ghost halos for the ghosts at coarse-fine boundaries to be correct.